### ISSUE TITLE
[CF 542] Condition: Spell Cooldown Forecast 

### ISSUE THREAD

**Issue by tmw-issue-import** [ascott18/TellMeWhen#564]
<p>Alright, so I've been thinking about this a while. As you know I use TellMeWhen to show what button I should be mashing next. Sometimes the condition of those icons is that another spell have a cooldown of at least 1 second remaining. This condition however is only reliable when the spell with the lower priority is ready.</p><p>How about implementing a condition that forecasts what the cooldown of another spell (b) will be once the cooldown of (a) is finished and then evaluates whether that number passes the condition? It would even be fairly easy. Just take the cooldown on (b) and subtract the cooldown on (a).</p><p>Now maybe this feature is too marginal for standalone implementation but maybe it's enough to justifiy a checkbox? Or maybe you could at least explain to me how to do this with a lua condition?</p> 
<br>

> Posted by CurseForge user <a href="https://wow.curseforge.com/members/AmeroGer">AmeroGer</a> | Imported from CurseForge issue <a href="https://wow.curseforge.com/projects/tellmewhen/issues/542">#542</a> | <a href="https://github.com/tmw-issue-import/dump/blob/master/html/542.html">Raw</a>
